frontend/src/components/activePage/TableInstance/TableInstance.tsx

```diff
@@ -38,7 +42,10 @@ const TableInstance: FC<ITableInstanceProps> = ({ ...props }) => {
   } = props;
 
   const [now, setNow] = useState(new Date());
-  const [deleteInstanceMutation] = useDeleteInstanceMutation();
+  const { createErrorCatcher } = useContext(ErrorContext);
+  const [deleteInstanceMutation] = useDeleteInstanceMutation({
+    onError: createErrorCatcher(ErrorTypes.ApolloError),
+  });
```

Comment:
```suggestion
  const [deleteInstanceMutation] = useDeleteInstanceMutation(withErrorCatcher());
```
Was thinking to make an even upper method that creates the { onError: ... } object itself at this point...

Reply:
OK but i'll have to add the generic type